src/utils/TypeConversions.res

```diff
@@ -0,0 +1,22 @@
+let float_to_RString = (input: float) : React.element => {
```

Comment:
Creating this function is not giving much value. Use `React.string("$"++input->Js.Float.toFixedWithPrecision(~digits=2))` wherever required